base/benchmarks/computation-BF16/cambricon/MLU/README.md

```diff
@@ -0,0 +1,44 @@
+# 参评AI芯片信息
+
+* 厂商：Cambricon
+
+## 服务器1
+
+- 产品名称：MLUXXX
+- 产品型号：MLU-XXX-XXXGB
```

Comment:
Use "/" as alternative. XXX seems like not developed yet

Reply:
have fixed it.